src/models/interfaces.ts

```diff
@@ -39,3 +39,29 @@ export interface AdoptReviewModel {
     adopt_review_editDate?: string;  // 후기 수정 날짜
     adopt_review_deleteDate?: string;// 후기 삭제 날짜
 }
+
+export interface MissingModel {
+    missing_id : number;    // 실종게시글 아이디
+    category_id : number;   // 카테고리 아이디
```

Comment:
category_id 가 1, 2만 있다면
```suggestion
    category_id : 1 | 2;   // 카테고리 아이디
```
로 하는 건 어떨까요? 그냥 제안입니당

Reply:
깔끔한 것 같습니다! 수정하였습니다~